accepted/0000-governance.md

```diff
@@ -0,0 +1,65 @@
+# Paketo Governance RFC
+
+This RFC outlines the govornance structure for the Paketo project.
+
+## Organizations:
+
+Paketo consists of two github organizations:
+
+- `paketo-buildpacks`
+- `paketo-community`
+
+The `paketo-buildpacks` github organization consists of actively supported buildpacks, libraries etc.
+
+The `paketo-community` github organization is designed as a staging ground for acceptance into the offically supported `paketo-buildpacks` org. Promotion from `paketo-community` into `paketo-buildpacks` requires an official [`rfc`](github.com/paketo-buildpacks/rfcs)
```

Comment:
Is paketo-community exclusively for staging things that before they go into paketo-buildpacks or are they just buildpacks that we are not including in paketo-buildpacks and therefore not in the official builder?